aip/general/0130.md

```diff
@@ -20,23 +21,24 @@ The following enumerates multiple categories of methods that exist, often
+| **Custom Methods**                 |                                                                                                                                                                                                               |                                                    |                                 |                 |                |                 |
+| Custom collection fetch methods    | Fetch information across a collection that cannot be expressed via a standard method.                                                                                                                         | [AIP-136][]                                        | automatable                     | automatable     | automatable    | automatable     |
+| Custom collection mutation methods | Perform an imperative operation referencing a collection that may mutate one or more resources within that collection in fashion that cannot be easily achieved by standard methods (e.g. state transitions). | [AIP-136][]                                        | handwritten                     | automatable     | automatable    | automatable     |
+| Custom resource fetch methods      | Fetch information for a single resource that cannot be expressed via a standard method.                                                                                                                       | [AIP-136][]                                        | automatable                     | automatable     | automatable    | automatable     |
+| Custom resource mutation methods   | Perform an imperative operation on a resource that may mutate it in a way a standard method cannot (e.g. state transitions).                                                                                  | [AIP-136][]                                        | handwritten                     | automatable     | automatable    | automatable     |
+| **None of the above**              |                                                                                                                                                                                                               |                                                    |                                 |                 |                |                 |
+| Streaming methods                  | Methods that communicate bi-directionally.                                                                                                                                                                    |                                                    | handwritten                     | handwritten     | handwritten    | automatable     |
```

Comment:
```suggestion
| Streaming methods                  | Methods that communicate via client, server, or bi-directional streams.                                                                                                                                                                   |                                                    | handwritten                     | handwritten     | handwritten    | automatable     |
```

Not that great of a suggestion, but at least it covers the bases? 😅

Reply:
agreed! we can always improve it: this AIP is in draft state for now.